### ISSUE TITLE
DATA/weights/vgg16.npy' not found

### ISSUE THREAD

**Issue by naserpiltan** [Chanakya-School-of-AI/KittiSeg#1]
Hello and thank you for this tutorial.
Im working on a road segmentation project with tensorflow.
I searched a lot and found this useful repo.
I cloned the repo and want to train it on the kitti dataset.

System information :
OS : ubuntu
Tensorflow version : 1.14.0

when i run the train command : **python train.py --hypes hypes/my_hype.json** 
It starts to download the kitti dataset and vgg16.npy . 
After successfully downloading those files , i have this error :

> 2020-06-19 14:04:40,833 INFO No environment variable 'TV_PLUGIN_DIR' found. Set to '/home/eagle-soft/tv-plugins'.
2020-06-19 14:04:40,834 INFO No environment variable 'TV_STEP_SHOW' found. Set to '50'.
2020-06-19 14:04:40,834 INFO No environment variable 'TV_STEP_EVAL' found. Set to '250'.
2020-06-19 14:04:40,834 INFO No environment variable 'TV_STEP_WRITE' found. Set to '1000'.
2020-06-19 14:04:40,834 INFO No environment variable 'TV_MAX_KEEP' found. Set to '10'.
2020-06-19 14:04:40,834 INFO No environment variable 'TV_STEP_STR' found. Set to 'Step {step}/{total_steps}: loss = {loss_value:.2f}; lr = {lr_value:.2e}; {sec_per_batch:.3f} sec (per Batch); {examples_per_sec:.1f} imgs/sec'.
WARNING:tensorflow:From train.py:131: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.
2020-06-19 14:04:40,834 WARNING From train.py:131: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.
2020-06-19 14:04:40,836 INFO f: <_io.TextIOWrapper name='hypes/KittiSeg.json' mode='r' encoding='UTF-8'>
I0619 14:04:40.836425 140663176697664 train.py:107] f: <_io.TextIOWrapper name='hypes/KittiSeg.json' mode='r' encoding='UTF-8'>
2020-06-19 14:04:40,870 INFO Downloading Kitti Road Data.
I0619 14:04:40.870168 140663176697664 kitti_seg_input.py:81] Downloading Kitti Road Data.
2020-06-19 14:04:40,870 INFO Download URL: https://s3.eu-central-1.amazonaws.com/avg-kitti/data_road.zip
I0619 14:04:40.870263 140663176697664 utils.py:38] Download URL: https://s3.eu-central-1.amazonaws.com/avg-kitti/data_road.zip
2020-06-19 14:04:40,870 INFO Download DIR: /media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA
I0619 14:04:40.870311 140663176697664 utils.py:39] Download DIR: /media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA
 ->**Downloading data_road.zip 100.0%**
2020-06-19 14:10:04,800 INFO Extracting kitti_road data.
I0619 14:10:04.800407 140663176697664 kitti_seg_input.py:84] Extracting kitti_road data.
2020-06-19 14:10:10,448 INFO Preparing kitti_road data.
I0619 14:10:10.448278 140663176697664 kitti_seg_input.py:88] Preparing kitti_road data.
2020-06-19 14:10:10,449 INFO Downloading VGG weights.
I0619 14:10:10.449632 140663176697664 kitti_seg_input.py:98] Downloading VGG weights.
2020-06-19 14:10:10,449 INFO Download URL: ftp://mi.eng.cam.ac.uk/pub/mttt2/models/vgg16.npy
I0619 14:10:10.449701 140663176697664 utils.py:38] Download URL: ftp://mi.eng.cam.ac.uk/pub/mttt2/models/vgg16.npy
2020-06-19 14:10:10,449 INFO Download DIR: /media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA
I0619 14:10:10.449749 140663176697664 utils.py:39] Download DIR: /media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA
-> **Downloading vgg16.npy 100.0%**
2020-06-19 14:15:50,779 INFO Initialize training folder
I0619 14:15:50.779425 140663176697664 train.py:124] Initialize training folder
2020-06-19 14:15:50,944 INFO Start training
I0619 14:15:50.944415 140663176697664 train.py:126] Start training
.
.
.
W0619 14:15:51.544675 140663176697664 deprecation.py:323] From /media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../inputs/kitti_seg_input.py:484: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
**2020-06-19 14:15:51,546 ERROR File '/media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA/weights/vgg16.npy' not found. Download it from ftp://mi.eng.cam.ac.uk/pub/mttt2/models/vgg16.npy
E0619 14:15:51.546804 140663176697664 fcn8_vgg.py:30] File '/media/eagle-soft/ExtDrive/segmentation/KittiSeg/hypes/../DATA/weights/vgg16.npy' not found. Download it from ftp://mi.eng.cam.ac.uk/pub/mttt2/models/vgg16.npy**

I checked it and i have vgg16.npy and kitti dataset in my project.
what is the problem ?
